scripts/php.sh

```diff
@@ -56,6 +60,7 @@ function run() {
 	cp -f /usr/share/doc/$VERSION-*/php.ini-development /etc/php.ini
 	sed -i "s/;date\.timezone.*/date\.timezone = ${TZ}/g" /etc/php.ini
 	sed -i "s/memory_limit.*/memory_limit = ${MEMORY_LIMIT}M/g" /etc/php.ini
+	sed -i "s/max_execution_time.*/max_execution_time = ${EXEC_TIME}/g" /etc/php.ini
```

Comment:
I think like #106 and as mentioned in #84, we should be echoing out the fact that we are potentially overwriting PHP settings here. Thoughts?